openbb_platform/openbb/assets/reference.json

```diff
@@ -2060,119 +2060,119 @@
                 "standard": [
                     {
                         "name": "underlying_symbol",
-                        "type": "str",
+                        "type": "List[str]",
```

Comment:
@deeleeramone disregarding the changes done in the `.py` files, shouldn't the reference look like this?

Reply:
It does. The reference file on `bugfix/security-updates` looks like this. However, it should still be showing as a ListUnion[str, None], and the default value should not be `""`. 

This would be an issue with Reference Generator, and not the model definitions though.

![Screenshot 2024-06-28 at 10 46 24 AM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/d45199cf-3a54-4527-98e5-46cdf7910903)